packages/material-ui/src/TextField/TextField.js

```diff
@@ -312,4 +313,4 @@ TextField.defaultProps = {
   variant: 'standard',
 };
 
-export default TextField;
+export default withStyles({}, { name: 'MuiTextField' })(TextField);
```

Comment:
What about having a root key so people can customize the UI too?

Reply:
I think we encountered this somewhere else too: `classes.root` vs. using the `className` prop?

Reply:
The advantage of my suggestion that people can do 
```jsx
const theme = createMuiTheme({
  overrides: {
    MuiTextField: {
      backgroundColor: '#ccc',
    },
  },
  props: {
    variant: "outlined",
  },
})
```

Reply:
Right. So `classes.root` is indeed preferred since it allows customization. Small nit: I believe it should be 
```diff
    MuiTextField: {
+    root: {
       backgroundColor: '#ccc',
+    }
    },
```

Reply:
Are you talking about `withStyles({ root: {} }, { name: 'MuiTextField' })(TextField);`?

Reply:
@oliviertassinari @eps1lon could you please provide more information about the `root` key as i am not sure what else should be done here. Thanks!

Reply:
@janowsiany I believe they are